telegram.ext.dispatcher - ERROR - No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "/home/maxim/.local/lib/python3.7/site-packages/telegram/ext/dispatcher.py", line 340, in process_update
    handler.handle_update(update, self, check, context)
  File "/home/maxim/.local/lib/python3.7/site-packages/telegram/ext/handler.py", line 119, in handle_update
    return self.callback(update, context)
  File "botik.py", line 28, in message_handler
    context.bot.send_message(chat_id = update.effective_message.chat_id, text = generate_message(message, chat_id))
  File "<decorator-gen-2>", line 2, in send_message
  File "/home/maxim/.local/lib/python3.7/site-packages/telegram/bot.py", line 67, in decorator
    result = func(*args, **kwargs)
  File "/home/maxim/.local/lib/python3.7/site-packages/telegram/bot.py", line 348, in send_message
    timeout=timeout, **kwargs)
  File "/home/maxim/.local/lib/python3.7/site-packages/telegram/bot.py", line 175, in _message
    result = self._request.post(url, data, timeout=timeout)
  File "/home/maxim/.local/lib/python3.7/site-packages/telegram/utils/request.py", line 333, in post
    **urlopen_kwargs)
  File "/home/maxim/.local/lib/python3.7/site-packages/telegram/utils/request.py", line 244, in _request_wrapper
    raise BadRequest(message)
telegram.error.BadRequest: Message text is empty
